airbyte-integrations/connectors/destination-clickhouse-v2/src/main/kotlin/io/airbyte/integrations/destination/clickhouse_v2/write/load/BinaryRowInsertBuffer.kt

```diff
@@ -63,28 +107,77 @@ class BinaryRowInsertBuffer(
                 .insert(
                     "`${tableName.namespace}`.`${tableName.name}`",
                     inner.toInputStream(),
-                    ClickHouseFormat.RowBinary
+                    ClickHouseFormat.RowBinary,
                 )
                 .await()
 
         log.info { "Finished insert of ${insertResult.writtenRows} rows into ${tableName.name}" }
     }
 
-    private fun writeAirbyteValue(columnName: String, abValue: AirbyteValue) {
+    internal fun writeAirbyteValue(
+        columnName: String,
+        abValue: AirbyteValue,
+        dataType: AirbyteType,
+    ) {
+        println(
```

Comment:
don't forget to remove this :)

Reply:
Done

Reply:
I think you maybe didn't commit this / didn't push this commit to github